Add troubleshooting cases for join issue (#289)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/troubleshooting/_index.md

### My device won't join. What do I do?

- Double check your DevEUI, JoinEUI or AppEUI, LoRaWAN and Regional Parameters Version, root keys (AppKey, and with LoRaWAN 1.1 or higher, NwkKey)
- Check gateway and device events for traffic from your device. Below are a few common issues

>     - **MIC mismatch** error in the device events: The possible cause is the mismatch of AppKey in device firmware to the AppKey registered in {{% tts %}} console. Update the AppKey in the console accordingly

Reviewer: ```suggestion
    - **MIC mismatch** error in the device events: Possible mismatch of AppKey in device firmware to the AppKey registered in {{% tts %}} console. Update the AppKey in the console accordingly
```

>     - **uplink channel not found** error in the gateway events: It indicates there is a mismatch of the frequency plans. Double check frequency plan settings in your end device and gateways (they must be the same LoRaWAN band)

Reviewer: ```suggestion
    - **Uplink channel not found** error in the gateway events: Indicates there is a mismatch of the frequency plans. Double check frequency plan settings in your end device and gateways (they must be the same LoRaWAN band)
```

>     - **DevNonce has already been used** error in the device events: It indicates a duplicate use of Devnonce. It generally happens when the device had sent too many unsuccessful Join Requests

Reviewer: ```suggestion
    - **DevNonce has already been used** error in the device events: Indicates a duplicate use of Devnonce. Generally happens when the device has sent too many unsuccessful Join Requests
```

>     - **uplink channel not found** error in the device events: It indicates the device is transmitting Join Requests in the non-default channels of the band which is not in line with the LoRaWAN Specification. We would suggest contacting the end-device manufacturer in this regard

Reviewer: ```suggestion
    - **Uplink channel not found** error in the device events: The device is transmitting Join Requests in non-default channels of the band which is not in line with the LoRaWAN Specification. Contact the end-device manufacturer
```
